src/sentry/static/sentry/app/components/charts/baseChart.jsx

```diff
@@ -54,6 +54,9 @@ class BaseChart extends React.Component {
     // Chart legend
     legend: SentryTypes.EChartsLegend,
 
+    // Chart legend truncate labels
+    legendTruncate: PropTypes.bool,
```

Comment:
hmmm, how about `truncateLegend` or something - I think making this a verb would be slightly better

Reply:
👍